nomad/mock/mock.go

```diff
@@ -18,7 +18,6 @@ func Node() *structs.Node {
 		Attributes: map[string]string{
 			"kernel.name":        "linux",
 			"arch":               "x86",
-			"nomad.version":      "0.5.0",
```

Comment:
What was the reason for removing this? Sorry, I remember you discussing it, but now I can't find it.

Reply:
It was because a fingerprint_manager for the white/blacklist would block the nomad fingerprinter from running then expect not to see the `nomad.version` key. I think at somepoint during the client refactor the mock.Node got setup in that test so it started failing

Reply:
Ah, interesting! Well as long as removing it didn't break anything that expected it to be setup. It's not clear to me what state the mock node is expected to be in (pre-fingerprinting? post-fingerprinting?), but that's nothing we can answer easily.